src/frontend/src/common/queries/groups.ts

```diff
@@ -77,6 +77,12 @@ export const mapGroupMemberData = (obj: RawGroupMemberRequest): GroupMember => {
   };
 };
 
+/**
+ * starting all group-related queries with "group" allows us to invalidate
+ * all of the queries in a single call to queryClient.invalidateQueries(["group"])
+ */
+export const GROUP_QUERY_PREFIX = "group";
+
```

Comment:
Alternatively we could call multiple times.  We await each one though, so this is a little faster.